### PR TITLE
SPARKC-285: replace *Type.setObject() with set() call

### PR DESCRIPTION
**Pull request by artem-aliev**

### CODE REVIEW

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TupleType.scala [hunk 1]
```diff
@@ -111,7 +112,7 @@ object TupleType {
         for (i <- 0 until fieldTypes.size) {
           val fieldConverter = fieldConverters(i)
           val fieldValue = fieldConverter.convert(tupleValue.getRaw(i))
-          toSave.setObject(i, fieldValue)
+          toSave.set(i, fieldValue,  CodecRegistryUtil.codecFor(fieldTypes(i), fieldValue))
```
Comment (jacek-lewandowski) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TupleType.scala]: formatting: 2 spaces before CodecRegistryUtil...

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/CodecRegistryUtil.scala [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.datastax.spark.connector.util
+
+import com.datastax.driver.core.{TypeCodec, CodecRegistry, DataType}
+
+object CodecRegistryUtil {
+  def codecFor(cqlType: DataType, value: AnyRef) : TypeCodec[AnyRef] = {
```
Comment (jacek-lewandowski) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/CodecRegistryUtil.scala]: formatting: colon right after closing bracket

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/CodecRegistryUtil.scala [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    if(value==null) CodecRegistry.DEFAULT_INSTANCE.codecFor(cqlType)
```
Comment (jacek-lewandowski) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/CodecRegistryUtil.scala]: formatting: `if (value == null)`

**Comment by jacek-lewandowski** [comment]
+1

**Comment by pkolaczk** [comment]
+1

**Pull request merged**